packages/nextjs-mf/src/plugins/container/InvertedContainerRuntimeModule.ts

```diff
@@ -1,11 +1,9 @@
-import {
-  Compiler,
-  RuntimeGlobals,
-  Template,
-  RuntimeModule,
-  Module,
-} from 'webpack';
+import { normalizeWebpackPath } from '@module-federation/sdk/normalize-webpack-path';
+import type { Module } from 'webpack';
```

Comment:
Should this use the normalize webpack path here?

Reply:
No, we only use RuntimeModule from here. Module is rarely used to extend from directly.